### ISSUE TITLE
Chat panel has contrast issues when using the night theme.

### ISSUE THREAD

**Issue by foldfree** [slskd/slskd#1017]
Night theme does not change the color of self messages, this is problematic for legibility purposes.
see:
<img width="1058" alt="Screenshot 2023-12-13 at 18 17 29" src="https://github.com/slskd/slskd/assets/34523150/3bc22c09-1a53-43d6-8890-30091d9ad9b8">
not WCAG compliant:
<img width="562" alt="Screenshot 2023-12-13 at 18 16 20" src="https://github.com/slskd/slskd/assets/34523150/247ee489-82a5-43d3-8cc0-6a1bd7558ece">

not APCA compliant:
<img width="1021" alt="Screenshot 2023-12-13 at 18 22 22" src="https://github.com/slskd/slskd/assets/34523150/99f0610a-638d-41d6-bd31-68ca420dd253">

suggestion:
use #add1ff instead

Why? I took the blue from iOS dark theme as a reference, and increased the brightness until [APCA Contrast Calculator](https://www.myndex.com/APCA/?BG=0d1116&TXT=0000ff&DEV=G4g&BUF=A22)
 say it is OK for body text.
<img width="1083" alt="Screenshot 2023-12-13 at 18 33 13" src="https://github.com/slskd/slskd/assets/34523150/d4becde4-9569-4219-a39d-4e4b38300e0c">

<img width="562" alt="Screenshot 2023-12-13 at 18 34 34" src="https://github.com/slskd/slskd/assets/34523150/409efb13-e2f7-4496-9703-8f89d07ba498">

**Comment by jpdillingham** [comment]
Thanks for reporting this, I noticed the other day and meant to create an issue to track it but forgot.

**Comment by jpdillingham** [comment]
I modeled this UI after SoulseekQT simply so I didn't have to put much thought into the design.  I'm thinking I can just change this up so it matches a modern chat interface, maybe like you'd find on Discord/Slack, or a left/right orientation like you find in Signal/Android SMS.

Thoughts?

**Comment by foldfree** [comment]
That would be great too:), actually much nicer.

Personally I prefer bubbles like signal. I find it much more legible.
 But maybe most people are expecting something like discord/slack.

**Comment by gsuberland** [comment]
In the meantime I've made some quick tweaks to the CSS to make the theme work a bit better in both themes. Fairly simple and should suffice until a more significant overhaul can be worked on.
